guest_image/run_vm.sh

```diff
@@ -1,21 +1,30 @@
-#!/bin/bash 
+#!/bin/bash
 
-VM_IMAGE=/home/baiksong/Work/giantvm/ubuntu1804-20G.img
+VM_IMAGE="bionic-server-cloudimg-amd64.img"
```

Comment:
What do you think if we can take the `VM_IMAGE` through an argument?
If the arg is empty, then set it to `bionic-server-cloudimg-amd64.img` as default.

For instance, 
1. ./run_vm.sh my.img: `VM_IMAGE` would be `my.img`
2.  ./run_vm.sh: `VM_IMAGE` would be `bionic-server-cloudimg-amd64.img`

Reply:
그냥 의견만 드린건데... 왜 리뷰어가 됐을까요... 푸는 방법 아시분 안계실까요..

Reply:
> 그냥 의견만 드린건데... 왜 리뷰어가 됐을까요... 푸는 방법 아시분 안계실까요..

review 자체도 contribution이니 괜찮지 않을까요?

Reply:
> What do you think if we can take the `VM_IMAGE` through an argument?
> If the arg is empty, then set it to `bionic-server-cloudimg-amd64.img` as default.

Yes, It will be more convenient for user. Plus, taking `NR_CPU`, `MEM_SIZE` and `GUI_MODE` through an argument will be save user's effort. 

I will apply it 

Thanks😄

Reply:
> 그냥 의견만 드린건데... 왜 리뷰어가 됐을까요... 푸는 방법 아시분 안계실까요..

review 자체도 contribution이니 괜찮지 않을까요🤔 ?

Reply:
That's a very good idea. I think it will be a good contribution if documents such as procedures and overviews are changed to mark down format.

Reply:
LGTM! Thank you for the contributions!